ga4gh/cli.py

```diff
@@ -181,6 +127,109 @@ def createGAListReferenceBasesRequest(self):
         return request
 
 
+def getWorkarounds(args):
+    if args.workarounds is None:
+        return set()
+    else:
+        return set(args.workarounds.split(','))
+
+
+##############################################################################
+# ga2vcf
+##############################################################################
+
+
+def addOutputFileArgument(parser):
+    parser.add_argument(
+        "--outputFile", "-o", default=None,
+        help="the file to write the output to")
+
+
+def ga2vcf_main(parser=None):
+    # parse args
+    if parser is None:
+        parser = argparse.ArgumentParser(
+            description="GA4GH VCF conversion tool")
+    addClientGlobalOptions(parser)
+    addOutputFileArgument(parser)
+    subparsers = parser.add_subparsers(title='subcommands',)
+    variantsSearchParser = addVariantsSearchParser(subparsers)
+    # TODO kinda ugly that we need to overload the parser with arguments
```

Comment:
We can note here that this is a temporary workaround until we have `getVariantSet` in the protocol.